gcp_airflow_foundations/source_class/generic_file_source.py

```diff
@@ -222,8 +222,8 @@ def load_to_landing(self, table_config, **kwargs):
             destination_path_prefix = gcs_bucket_prefix + table_name + "/" + date
             logging.info(destination_path_prefix)
 
-        files_to_load = [destination_path_prefix + "/" + f for f in files_to_load]
-        logging.info(files_to_load)
+            files_to_load = [destination_path_prefix + "/" + f for f in files_to_load]
```

Comment:
wouldn't the "/" cause the same issue as with gcs_bucket_prefix?

Reply:
destination_path_prefix should never be empty unlike gcs_bucket_prefix since it also includes the table_name and the date. 

Additionally this code is only called if the gcs_bucket_path_format_mode is passed in the config